tests/src/integration/docker/actions/go/sample.go

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Do we need the source code for the "exec" binary?

Reply:
This is a sample in Go from https://console.bluemix.net/docs/openwhisk/openwhisk_actions.html#creating-go-actions. I have three examples included, (1) a dir with exec in bash (function: <dir>) (2) a dir with exec in go (compiled) (function: <dir>) (3) exec.zip. Its pretty much same workflow for example.c